content/index.js

```diff
@@ -82,6 +82,14 @@ const transformData = (
     return autoEntries.length === 1 ? autoEntries[0] : autoEntries
   }
 
+  if (strategy === 'object') {
+    return processedEntries.length === 1
+      ? processedEntries[0][1]
+      : Object.fromEntries(
+          processedEntries.map(([key, value]) => [camelize(key), value])
+        )
```

Comment:
It's good you found this. Looks much simpler, doesn't it?

When you look above, the `camelize` is called on the value. Here you don't do this on the value preserving original keys in the object (when the `value` is an object). It's unexpected behaviour, the value must be camelized as well.

Camelizing keys and values nothing else than camelizing object created from keys and values.

```suggestion
      : camelize(Object.fromEntries(processedEntries))
```

Reply:
It indeed looks much simpler